### ISSUE TITLE
Android New Fingerprint Added

### ISSUE THREAD

**Issue by cvlehe** [naoufal/react-native-touch-id#168]
I don't see a way to detect if the user has added a new fingerprint to the device since logging in. This is a potential security issue as anyone can access a previously logged in account by adding a new fingerprint to the device.

**Comment by tonmanayo** [comment]
@cvlehe did you manage to solve this issue?

**Comment by cvlehe** [comment]
You basically need to fork the repo and modify react-native-touch-id to pass back an error saying there was a new fingerprint enrolled. I'm going to submit a PR in the next couple of days with this feature.

**Comment by zibs** [comment]
Thanks!

**Comment by tonmanayo** [comment]
I don't think that will solve the issue. You need to return some kinda ID that is linked with the finger print and save it with the users profile. At moment it only returns true
